.github/ISSUE_TEMPLATE/create-a-test.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Create a Test
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the feature you want to create a test**
+A clear and concise description of what the test is.
+Ex.
+```
+As a [role]
+I want [feature]
+So that [benefit]
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```

Comment:
idk if this is relevant here? all tests should by default be cross browser compatible